cmd/notary/keys.go

```diff
@@ -338,7 +365,56 @@ func (k *keyCommander) keysImportRoot(cmd *cobra.Command, args []string) error {
 	}
 	defer importFile.Close()
 
-	err = cs.ImportRootKey(importFile)
+	pemBytes, err := ioutil.ReadAll(importFile)
+	if err != nil {
+		return fmt.Errorf("Error reading input file: %v", err)
+	}
+
+	pemRole := trustmanager.ReadRoleFromPEM(pemBytes)
+
+	// Rewind after reading the first time
+	_, err = importFile.Seek(0, 0)
```

Comment:
This was to keep the filepointer at the beginning of the file for `ImportRootKey` but I think I'll follow @cyli's suggestion and refactor `ImportRoleKey`.  That should get rid of this